Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -97,6 +97,16 @@ Create a custom command
 See the :ref:`Tutorial: Create a console command <console-command-tutorial>`
 for details on how to create commands.
 
+DataHandler usage
+=================
+Using the :ref:`DataHandler <tce-database-basics>` in a cli command will
+fail with the following error:
```

Comment:
Maybe better "with an error like this", because other Tables may be affected.

Also I think we usually uppercase "CLI".

(Sidenote: is the error rendered with special markup, do we need a special admonition for it?)